File: Java/1120_revisao_contrato.java
import java.util.Scanner;

class Main {
	public static void main(String[] args) {
		System.out.println("Aôba");
		Scanner scanner = new Scanner(System.in);

		String line = scanner.nextLine();

		while (!line.equals("0 0")) {
			String[] splittedLine = line.split(" ");
			String missingLetter = splittedLine[0];
			String numberToCheck = splittedLine[1];
			String answer = new String();

			Boolean skipZero = true;

			for (int i = 0; i < numberToCheck.length(); i++) {
				if (skipZero) {
					if ()
				}
			}

			line = scanner.nextLine();
		}
	}
}

File: Javascript/1120_revisao_contrato.js
const readline = require("readline");

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout,
});

rl.on("line", (answer) => {
  if (answer === "0 0") {
    rl.close();
    return;
  }
  let s = answer.split(" ");
  let typed_number = s[1];
  let missing_letter = s[0];
  let result = "";
  let dont_accept_zero = true;
  for (let i = 0; i < typed_number.length; i++) {
    if (dont_accept_zero) {
      if (typed_number[i] !== "0" && typed_number[i] !== missing_letter) {
        result += typed_number[i];
        dont_accept_zero = false;
      }
    } else {
      if (typed_number[i] !== missing_letter) {
        result += typed_number[i];
      }
    }
  }
  if (result === "") result = "0";
  console.log(result);
});
